src/Nethermind/Nethermind.Consensus.AuRa/Validators/ReportingContractBasedValidator.PersistentReports.cs

```diff
@@ -118,7 +119,7 @@ private void FilterReports(BlockHeader parent)
                         if (_logger.IsTrace) _logger.Trace("Successfully removed report from report cache.");
                     }
                 }
-                catch (AuRaException e)
+                catch (AbiException e)
```

Comment:
in the other file you change AbiException to AuRa and here you changed the exception that you catch from AuRa to Abi. Was it intended?